### PR TITLE
Fix ExpoFontLoader on Internet Explorer

### LINKED ISSUE #6752: Expo vector icons are broken on IE11
This is the same issue as #5935 with the same root cause, only the browser is different (IE11 instead of Edge)

### PR DESCRIPTION
**Pull request by d4rky-pl**
This PR adds IE exclusion to `isFontLoadingListenerSupported` which fixes font loader on IE11

# Why

Fixes #6752 

# Test Plan

- Import MaterialIcons from @expo/vector-icons and try to render one
- Open the page in Internet Explorer 11

### CODE REVIEW

**Review by sjchmiela — COMMENTED** [review]
Thank you very much for taking care of this! Looks perfect! 😍 

One question before we merge it — could you please add this change to `packages/expo-font/CHANGELOG.md` and run `yarn` in `packages/expo-font` so that updated `build` folder also is included in the PR? 🙏

File: packages/expo-font/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Fixed timeout on IE [#8652](https://github.com/expo/expo/pull/8652)
```
Comment (sjchmiela) [inline packages/expo-font/CHANGELOG.md:11]: Does it fix fonts not being loaded or fix timeout when loading fonts (they still don't load even so)?
```suggestion
- Fixed fonts not being loaded in Internet Explorer ([#8652](https://github.com/expo/expo/pull/8652) by [@d4rky-pl](https://github.com/d4rky-pl))
```

**Review by sjchmiela — APPROVED** [review]
Looks good to me! 🙇

File: packages/expo-font/CHANGELOG.md (hunk 1 above)
Comment (d4rky-pl) [reply packages/expo-font/CHANGELOG.md:11]: Fixes both loading and the timeout I guess (they load and don't timeout). I based the message on the same one but for Firefox (which uses exactly the same fix)

Comment (sjchmiela) [reply packages/expo-font/CHANGELOG.md:11]: Yeah, that change log has been introduced after the actual fix as a history log, the common format we use is something more like the text proposed, eg.

https://github.com/expo/expo/blob/52217f1a59f2d02ebbb4eeeea3e24064cd544980/packages/expo-notifications/CHANGELOG.md#L39

**Comment by sjchmiela** [comment]
Thank you very much for this contribution! You've done a great job! 💯

**Pull request merged**